collector/library/stats.js

```diff
@@ -17,24 +19,25 @@ const stats = {
         for (let n in this.pods) {
             let pod = this.pods[n];
 
-            if (!logstashConnected) {
-                logstashConnected = !pod.logstashConnectivityReported;
-            }
-
             totalSentBytes += pod.stats.totalSentBytes;
             totalUnackedBytes += pod.stats.totalUnackedBytes;
         }
 
+        if (totalSentBytes > lastSentBytes){
```

Comment:
what are we doing with `totalUnackedBytes`? this number cannot be interesting because we never send more than one line - if it fails, we start the entire flow anew. if it succeeds this number comes back to zero. so it will always be some small value or zero - consider deleting this variable from memory and avoid printing it